components/HamburgerMenu.vue

```diff
@@ -307,7 +336,12 @@ onMounted(() => {
     }
 })
 
-//this is a custom directive to close the menu when clicking outside of the menu
+// converts the theme name to locale case (first letter uppercase, rest lowercase)
+function toLocaleCase(theme: string) {
```

Comment:
I think this should be Pascal Case instead of Locale
```suggestion
function toPascalCase(theme: string) {
```

Also, this would likely be a great additon to move into our utils so it can be reused 😸

Reply:
moved to stringUtils.ts and renamed in forthcoming commit.

Reply:
oof, you're right! I think I mean't Sentencecase, not PascalCase 😆  this looks good now 👍